playbooks/roles/configure_lbaasv2_agent/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+undercloud: true
```

Comment:
We've been conflating over and under cloud with GRM or L2 Adjacent Mode.  Rename this with "f5_global_routed_mode" which can be use directly to populate the ini file.